src/frontend/src/common.tsx

```diff
@@ -974,6 +974,7 @@ export const icrcTransfer = async (
     } catch (e) {
         return "Transfer failed";
     }
+    return response;
```

Comment:
This is wrong, this variable is undefined here.

Reply:
My bad.